Drag-n-Drop.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE HTML>
+<html>
+	<!--head starts-->
+	<head>
+		<title>Drag n Drop</title>
+		<meta name = "author" content = "Akshita Dixit">
+		<!--styling for the web page-->
+		<style>
+			#div1, #div2 {
+				float: left;
+				width: 200px;
+				height: 100px;
+				margin: 10px;
+				padding: 10px;
+				border: 1px solid black;
+			}
+		</style>
+		<!--script for the web page-->
+		<script>
+			function allowDrop(ev) {

```

Comment:
Scripts should be placed after the body, before the closing `<html>` tag, so as to allow for the document to be ready before it is processed by the script.